cql/src/main/java/io/stargate/cql/CqlActivator.java

```diff
@@ -66,19 +66,17 @@ private synchronized void maybeStartService(
     if (cql != null) { // Already started
       return;
     }
-
-    cql = new CqlImpl(makeConfig());
-    cql.start(persistence, metrics, authentication);
+    cql = new CqlImpl(makeConfig(), persistence, metrics, authentication);
     log.info("Starting CQL");
+    cql.start();
   }
 
   private synchronized void maybeStopService() {
-    CqlImpl c = cql;
-    if (c != null) {
-      log.info(("Stopping CQL"));
-      c.stop();
+    if (cql != null) { // Not started
```

Comment:
how do we know that it is not started in that case? I would suspect that if `cql == null` then not started.